rules/cloud/azure/azure_ad_risky_sign_ins_with_singlefactorauthencation_from_unknown_devices.yml

```diff
@@ -0,0 +1,24 @@
+title: Suspicious Sign-Ins from a Non Registered Device
+id: 572b12d4-9062-11ed-a1eb-0242ac120002
+status: experimental
+description: Detects risky authencaition from a non AD registered device without MFA being required.
+references:
+    - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-devices#non-compliant-device-sign-in
```

Comment:
Can you point me to where in the reference you inspired the logic? I can't seem to find it. Thanks in advance.